atom/browser/importer/profile_writer.cc

```diff
@@ -170,31 +173,62 @@ void ProfileWriter::AddAutofillFormDataEntries(
   }
 }
 
+// Called when SetCookieWithDetailsAsync completes
+static void OnSetCookie(const ImportedCookieEntry& cookie, bool success) {
+  if (!success) {
+    LOG(ERROR) << "importing cookie for "
```

Comment:
I think we need to provide some kind of notification back to browser-laptop since in most cases no one will ever see this log or know there was a problem